src/Microsoft.SqlTools.ServiceLayer/Profiler/XEventSession.cs

```diff
@@ -14,6 +14,11 @@ public class XEventSession : IXEventSession
     {
         public Session Session { get; set; }
 
+        public void Drop()
+        {
+            this.Session.Drop();
```

Comment:
Do we want to drop the session when the user clicks `Stop` or should we just stop the sessions?  I think SSMS only stops the session in this scenario.